sickbeard/server/web/config/notifications.py

```diff
@@ -258,7 +258,7 @@ def saveNotifications(self, use_kodi=None, kodi_always_on=None, kodi_notify_onsn
 
         sickbeard.save_config()
 
-        if len(results) > 0:
+        if results:
```

Comment:
Nearest I could tell, nowhere (its assigned an empty list that does not appear to be used) so this code may be able to be removed.  However, I figured it would be premature to remove it until thats confirmed, so I have left that for later optimization.